### PR TITLE
feature---resource-listing

### PR DESCRIPTION
**Pull request by dbeau79**

### CODE REVIEW

File: theme-beacon/css/05-components/_resources.css [hunk 1]
```diff
@@ -89,6 +103,11 @@
   color: {{ cream }}
 }
 
+.tile--resource .btn {
+  padding-left: 15px;
```
Comment (danaholdt) [inline theme-beacon/css/05-components/_resources.css:107]: Hey @dbeau79 ! You think we could achieve this padding using the bootstrap in place of adding an additional class for the button?

Comment (dbeau79) [reply theme-beacon/css/05-components/_resources.css:107]: @danaholdt do you mean adding a class to the button like "px-2" for example?

Comment (danaholdt) [reply theme-beacon/css/05-components/_resources.css:107]: yep!

**Comment by danaholdt** [comment]
@dbeau79 Im merging this to let you move forward. We can try this experiment again tomorrow 👍

**Pull request merged**